public/locales/de/invitationModal.json

```diff
@@ -0,0 +1,6 @@
+{
+  "was invited and needs to accept your invitation": "wurde eingeladen und muss die Einladung annehmen",
```

Comment:
Please put the user / name variable inside the string. In the case of German & English it might work to assume that it always comes at the beginning of the sentence.
But in other languages that might not be the case. So it's always good practice to include everything.